server/events/command_runner.go

```diff
@@ -134,55 +134,89 @@ func (c *DefaultCommandRunner) RunAutoplanCommand(baseRepo models.Repo, headRepo
+		if statusErr := c.CommitStatusUpdater.UpdateCombined(ctx.Pull.BaseRepo, ctx.Pull, models.FailedCommitStatus, cmdModel); statusErr != nil {
 			ctx.Log.Warn("unable to update commit status: %s", statusErr)
 		}
 
-		c.updatePull(ctx, AutoplanCommand{}, CommandResult{Error: err})
-		return
+		c.updatePull(ctx, pullCommand, CommandResult{Error: err})
+		return err
 	}
+
 	if len(projectCmds) == 0 {
-		log.Info("determined there was no project to run plan in")
+		ctx.Log.Info("determined there was no project to run %s in", cmdModel.String())
 		if !c.SilenceVCSStatusNoPlans {
 			// If there were no projects modified, we set a successful commit status
 			// with 0/0 projects planned successfully because some users require
 			// the Atlantis status to be passing for all pull requests.
 			ctx.Log.Debug("setting VCS status to success with no projects found")
-			if err := c.CommitStatusUpdater.UpdateCombinedCount(baseRepo, pull, models.SuccessCommitStatus, models.PlanCommand, 0, 0); err != nil {
+			if err := c.CommitStatusUpdater.UpdateCombinedCount(ctx.Pull.BaseRepo, ctx.Pull, models.SuccessCommitStatus, cmdModel, 0, 0); err != nil {
 				ctx.Log.Warn("unable to update commit status: %s", err)
 			}
 		}
-		return
+		return err
 	}
 
 	// At this point we are sure Atlantis has work to do, so set commit status to pending
-	if err := c.CommitStatusUpdater.UpdateCombined(ctx.Pull.BaseRepo, ctx.Pull, models.PendingCommitStatus, models.PlanCommand); err != nil {
+	if err := c.CommitStatusUpdater.UpdateCombined(ctx.Pull.BaseRepo, ctx.Pull, models.PendingCommitStatus, cmdModel); err != nil {
 		ctx.Log.Warn("unable to update commit status: %s", err)
 	}
 
 	// Only run commands in parallel if enabled
 	var result CommandResult
-	if c.parallelPlanEnabled(ctx, projectCmds) {
+	switch {
+	case cmdModel == models.PlanCommand && c.parallelPlanEnabled(ctx, projectCmds):
 		ctx.Log.Info("Running plans in parallel")
-		result = c.runProjectCmdsParallel(projectCmds, models.PlanCommand)
-	} else {
-		result = c.runProjectCmds(projectCmds, models.PlanCommand)
+		result = c.runProjectCmdsParallel(projectCmds, cmdModel)
+	case cmdModel == models.PolicyCheckCommand && c.parallelPolicyCheckEnabled(ctx, projectCmds):
+		ctx.Log.Info("Running policy checks in parallel")
+		result = c.runProjectCmdsParallel(projectCmds, cmdModel)
+	default:
+		result = c.runProjectCmds(projectCmds, cmdModel)
 	}
 
 	if c.automergeEnabled(ctx, projectCmds) && result.HasErrors() {
 		ctx.Log.Info("deleting plans because there were errors and automerge requires all plans succeed")
 		c.deletePlans(ctx)
 		result.PlansDeleted = true
 	}
-	c.updatePull(ctx, AutoplanCommand{}, result)
+	c.updatePull(ctx, pullCommand, result)
 	pullStatus, err := c.updateDB(ctx, ctx.Pull, result.ProjectResults)
 	if err != nil {
 		c.Logger.Err("writing results: %s", err)
 	}
 
-	c.updateCommitStatus(ctx, models.PlanCommand, pullStatus)
+	c.updateCommitStatus(ctx, cmdModel, pullStatus)
```

Comment:
Do you need to update this function?
https://github.com/lyft/atlantis/blob/ORCA-470-add-policy-check-step-runner/server/events/command_runner.go#L380:L381